4_params_munge.yml

```diff
@@ -97,7 +97,7 @@ targets:
 
   4_params_munge/out/lagos_depths.rds.ind:
     command: munge_lagos_depths(target_name,
-      lagos_depths_ind = '3_params_fetch/out/LAGOS_NE_depths.csv.ind',
+      lagos_depths_ind = '3_params_fetch/in/LAGOS_US_depths.csv.ind',
       lagos_xwalk_ind = '2_crosswalk_munge/out/lagosne_nhdhr_xwalk.rds.ind')
```

Comment:
The rest of this PR looks 👍 but you need to create a new lagos crosswalk (preferably, scdel the prior one) for the national dataset. Since you are still using the LAGOS_NE xwalk, you aren't going to get new lakes that are outside of that original footprint. 

But, I think the new crosswalk will be easier than the prior one because you won't need to do a poly-to-poly match, since the LAGOS-US metadata file has an NHDHR field, so all you would need to do is prefix that with `nhdhr_`, rename the other field for LAGOS, and drop the rest of the fields/columns. Happy to explain this in more detail, but you should be able to drop a bunch of LAGOS-NE targets because the new xwalk is simpler than the prior one.

Reply:
More info for you here:

The `lake_information.csv` file in the LAGOS-US LOCUS module provides lagos IDs (see `lagoslakeid`) _and_ NHDHR ids (see `lake_nhdid`):
```r
> read_csv('~/Downloads/lake_information.csv')
Rows: 479950 Columns: 34                                                                                                                                                              
── Column specification ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
Delimiter: ","
chr (24): lake_nhdid, lake_reachcode, lake_namegnis, lake_namelagos, lake_onlandborder, lake_ismultipart, lake_missingws, lake_shapeflag, lake_...
dbl (10): lagoslakeid, lake_nhdfcode, lake_nhdftype, lake_lat_decdeg, lake_lon_decdeg, lake_elevation_m, buff100_zoneid, buff500_zoneid, ws_zon...

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.
# A tibble: 479,950 × 34
   lagoslakeid lake_nhdid lake_nhdfcode lake_nhdftype lake_reachcode lake_namegnis  lake_namelagos lake_onlandbord… lake_ismultipart lake_missingws
         <dbl> <chr>              <dbl>         <dbl> <chr>          <chr>          <chr>          <chr>            <chr>            <chr>         
 1        8056 142826768          39004           390 01010006004479 Saint Francis… Saint Francis… N                N                N             
 2       19412 142806711          39004           390 01010002002046 Pillsbury Pond Pillsbury Pond N                Y                N             
 3        7956 142978929          39004           390 01010003000805 Fish River La… Fish River La… N                Y                N             
 ```
 
 That file, other not immediately relevant files, and the citation information is found [here](https://doi.org/10.6073/pasta/e5c2fb8d77467d3f03de4667ac2173ca)

Reply:
There are 461 lakes I am seeing from the LAGOS_NE xwalk that don't have the same LAGOS_ID + NHDHR ID in the LAGOS_US xwalk

![image](https://user-images.githubusercontent.com/13220910/163879105-80d8fd3e-4776-440f-88f4-483ff4ec6750.png)

```r
library(scipiper)
library(tidyverse)

scmake('8_viz/out/lakes_summary_fig.html.ind')

old_xwalk <- readRDS(sc_retrieve('2_crosswalk_munge/out/lagosne_nhdhr_xwalk.rds.ind'))
new_xwalk <- readr::read_csv('lake_information.csv') %>%
  mutate(LAGOS_ID = sprintf("lagos_%s", lagoslakeid),
         site_id = sprintf("nhdhr_%s", lake_nhdid)) %>% 
  dplyr::select(LAGOS_ID, site_id) 
lakes_sf <- readRDS(sc_retrieve('1_crosswalk_fetch/out/canonical_lakes_sf.rds'))

new_appear_in_old <- semi_join(new_xwalk, old_xwalk) 
old_appear_in_new <- semi_join(new_xwalk, old_xwalk) 
old_missing_in_new <- anti_join(old_xwalk, old_appear_in_new)

maps::map('state', tolower(c("North_Dakota","South_Dakota","Iowa","Michigan",
                             "Indiana","Illinois","Wisconsin", "Minnesota",
                             "New_York", "Pennsylvania", "Missouri", "Arkansas")))
lakes_sf %>% 
  filter(site_id %in% old_missing_in_new$site_id) %>% 
  st_geometry() %>% 
  plot(add=TRUE)
```

Reply:
I think LAGOS NE was based on NHD medium res polygons (similar to Luke's original data paper) so I think some changes are expected here even in the overlapping region. :+1: